Update ethics-soc-6.md (#2165)
huggingface/blog: ethics-soc-6.md

## Data Quality Practices at Hugging Face

> Ensuring high data quality is essential for developing effective and reliable AI models. Here are some examples of the several data quality strategies that we employ at Hugging Face:

Reviewer: ```suggestion
Ensuring high data quality is essential for developing effective and reliable AI models. Here are some examples of data quality strategies that teams at Hugging Face have employed:
```
